WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -15,29 +15,196 @@ struct AddEditCoupon: View {
+            GeometryReader { geometry in
+                ScrollView {
+                    VStack (alignment: .leading) {
+                        Group {
+                            ListHeaderView(text: Localization.headerCouponDetails.uppercased(), alignment: .left)
 
-            //TODO: implement the content of the view
-            Text("Hello, World!")
-                .toolbar {
-                                    ToolbarItem(placement: .cancellationAction) {
-                                        Button("Cancel", action: {
-                                            presentation.wrappedValue.dismiss()
-                                        })
-                                    }
-                                }
+                            Group {
+                                TitleAndTextFieldRow(title: Localization.couponAmountPercentage,
+                                                     placeholder: Localization.couponAmountPercentage,
+                                                     text: $viewModel.amountField,
+                                                     editable: false,
+                                                     fieldAlignment: .leading,
+                                                     keyboardType: .asciiCapableNumberPad)
+                                Divider()
+                                    .padding(.leading, Constants.margin)
+                            }
+
+                            Text(Localization.footerCouponAmountPercentage)
+                                .subheadlineStyle()
+                                .padding(.horizontal, Constants.margin)
+
+                            Group {
+                                TitleAndTextFieldRow(title: Localization.couponCode,
+                                                     placeholder: Localization.couponCode,
+                                                     text: $viewModel.codeField,
+                                                     editable: false,
+                                                     fieldAlignment: .leading,
+                                                     keyboardType: .asciiCapableNumberPad)
+                                Divider()
+                                    .padding(.leading, Constants.margin)
+                            }
+
+                            Text(Localization.footerCouponCode)
+                                .subheadlineStyle()
+                                .padding(.horizontal, Constants.margin)
+
+                            //TODO: leading aligning for this button
+                            Button {
+                                //TODO: handle action
+                            } label: {
+                                Text(Localization.regenerateCouponCodeButton)
+                            }
+                            .buttonStyle(LinkButtonStyle())
+                            .padding(.horizontal, Constants.margin)
```

Comment:
I think the button is left-aligned in the design - is it possible to make it align to the left instead of center?

<img width="300" alt="Screen Shot 2022-04-04 at 17 28 03" src="https://user-images.githubusercontent.com/5533851/161525733-919fe0d3-d822-4661-bd84-07571591342f.png">